### PR TITLE
ansible: improve keystone supports, create skydive user and tripleo job

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run devstack

**Comment by safchain** [comment]
run devstack-tests

File: scripts/ci/tripleo-quickstart/minimal.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# This is a basic configuration it should match fs001 as
+# closely as possible with the exception of the network isolation type
+
+# Deploy an Openstack environment with ssl undercloud, introspect, and use network isolation
+# Note: any change in this featureset must also be done in featureset035.
+
+# This enables TLS for the undercloud which will also make haproxy bind to the
+# configured public-vip and admin-vip.
+overcloud_templates_path: "{{ working_dir }}/tripleo-heat-templates"
+overcloud_templates_repo: https://github.com/safchain/tripleo-heat-templates
```
Comment (safchain) [inline scripts/ci/tripleo-quickstart/minimal.yml]: need to be moved as extra_vars

**Comment by lebauce** [comment]
@safchain It seems you forgot scripts/ci/run-tripleo-tests.sh

**Comment by safchain** [comment]
@lebauce added

**Comment by safchain** [comment]
run tripleo-tests

**Comment by safchain** [comment]
run tripleo-tests

**Comment by safchain** [comment]
run tripleo-tests

**Comment by safchain** [comment]
run tripleo-tests

**Comment by safchain** [comment]
run tripleo-tests

**Comment by safchain** [comment]
run devstack-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by safchain** [comment]
rerun tripleo-tests

**Comment by lebauce** [comment]
run vagrant-tests

**Comment by safchain** [comment]
run vagrant-tests

**Pull request merged**